src/interfaces/database.ts

```diff
@@ -227,3 +227,20 @@ export interface JobModel extends Document {
   isInternship?: boolean;
   platform: string;
 }
+
+export interface Action {
```

Comment:
Our codebase is getting huge. Don't name something so simple or one word. Define it properly. like UserPointsAction